front/lib/swr.ts

```diff
@@ -407,6 +408,21 @@ export function usePokeFeatures({ workspaceId }: { workspaceId: string }) {
   };
 }
 
+export function useFeatures({ workspaceId }: { workspaceId: string }) {
```

Comment:
let's take the owner: WorkspaceType as argument ?

Reply:
ok